### ISSUE TITLE
Doesn't work on M2 macbook pro

### ISSUE THREAD

**Issue by farhansaed** [xen0l/aws-gate#1598]
MacOs : vantura 13.2.1
Installation went fine however running aws-gate gives below error.

```
Traceback (most recent call last):
  File "/opt/homebrew/bin/aws-gate", line 3, in <module>
    import aws_gate.cli
  File "/opt/homebrew/Cellar/aws-gate/0.11.2/libexec/lib/python3.11/site-packages/aws_gate/cli.py", line 25, in <module>
    from aws_gate.exec import exec
  File "/opt/homebrew/Cellar/aws-gate/0.11.2/libexec/lib/python3.11/site-packages/aws_gate/exec.py", line 4, in <module>
    from aws_gate.decorators import (
  File "/opt/homebrew/Cellar/aws-gate/0.11.2/libexec/lib/python3.11/site-packages/aws_gate/decorators.py", line 7, in <module>
    from wrapt import decorator
  File "/opt/homebrew/Cellar/aws-gate/0.11.2/libexec/lib/python3.11/site-packages/wrapt/__init__.py", line 10, in <module>
    from .decorators import (adapter_factory, AdapterFactory, decorator,
  File "/opt/homebrew/Cellar/aws-gate/0.11.2/libexec/lib/python3.11/site-packages/wrapt/decorators.py", line 34, in <module>
    from inspect import ismethod, isclass, formatargspec
ImportError: cannot import name 'formatargspec' from 'inspect' (/opt/homebrew/Cellar/python@3.11/3.11.2/Frameworks/Python.framework/Versions/3.11/lib/python3.11/inspect.py)

```

**Comment by larskoe** [comment]
I have the same issue! Also on an M2 Pro MacBook Pro.

**Comment by jvdrean** [comment]
I opened a PR which (as far as I can tell) fixes the issue :) 
https://github.com/xen0l/aws-gate/pull/1601

**Comment by jlongman** [comment]
does this just need to be released?

**Comment by jvdrean** [comment]
@jlongman I think so yes

**Comment by maku693** [comment]
I have the same problem. @xen0l Do you have a plan to release a new version any soon?

**Comment by xen0l** [comment]
Hey people, sorry for the delay. I just released a new version `0.11.3` and would like you to test it, especially on Macs with M1/M2 hardware.

**Comment by maku693** [comment]
Thank you! I've tested it on M2 MacBook Pro. Both pip and homebrew installation works properly.
